src/components/Ticket.js

```diff
@@ -0,0 +1,53 @@
+import {formatTime} from "../helpers/formatTime.js";
+import {getCompanyLogo} from "../helpers/getCompanyLogo";
+import {useSegments} from "../hooks/useSegments";
+
+export function Ticket(props){
```

Comment:
Деструктуризация для пропсов
```suggestion
const Ticket = ({price, companyId, segments}) => {...}
```